sources/europe/ch/Kanton_Glarus_ortho2013.geojson

```diff
@@ -0,0 +1,101 @@
+{
+    "type": "Feature",
+    "bbox": "",
```

Comment:
Hello! It looks like you have extra bbox attributes that are breaking the tests. Can you please remove them?